FeedStoreChallenge/Infrastructure/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Fernando Goulart on 01/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+internal class ManagedFeedImage: NSManagedObject {
+	@NSManaged internal var id: UUID
+	@NSManaged internal var imageDescription: String?
+	@NSManaged internal var location: String?
+	@NSManaged internal var url: URL
+	@NSManaged internal var cache: ManagedCache
```

Comment:
`internal` is a default access level, no need to declare it explicitly.
Also `cache` is optional in the model description in `FeedStore.xcdatamodel`. 

Please make `cache` optional here so that the class declaration and the model description are consistent and remove `internal` modifier.

```suggestion
	@NSManaged var cache: ManagedCache?
```